src/components/course-calendar/plan-details.cjsx

```diff
@@ -37,6 +37,9 @@ CoursePlanDetails = React.createClass
     editLinkName = camelCase("edit-#{type}")
 
     reviewButton = @renderReviewButton()
+    editButton = <Router.Link to={editLinkName} params={linkParams}>
+      <BS.Button>Edit Assignment</BS.Button>
+    </Router.Link> if plan.isEditable
 
```

Comment:
I've struggled with the cleanest way to write conditional jsx with coffeescript.  The best I've found thus far is:

```coffee
    if plan.isEditable then editButton =
      <Router.Link to={editLinkName} params={linkParams}>
        <BS.Button>Edit Assignment</BS.Button>
      </Router.Link>
```

You can also put a `\` after the `if` line to maybe make it more clear that it's a continuation.
i.e. ` if plan.isEditable then editButton = \`  I'm not sure if that adds much or not though.